radicle/src/identity/project.rs

```diff
@@ -402,6 +402,22 @@ pub struct Identity<I> {
     pub signatures: HashMap<PublicKey, Signature>,
 }
 
+impl radicle_cob::identity::Identity for Identity<Oid> {
+    type Identifier = Oid;
+
+    fn is_delegate(&self, delegation: &crypto::PublicKey) -> bool {
+        self.doc.delegates.iter().any(|d| d.matches(delegation))
+    }
+
+    fn content_id(&self) -> Oid {
+        self.current
+    }
+
+    fn identifier(&self) -> Self::Identifier {
+        self.root
+    }
+}
```

Comment:
Interesting :O

Reply:
... that both this type and `Author` are impls of `Identity`, given that this is a full doc while `Author` is just a pubkey. Something seems slightly off with regards to the intent of this trait, but I would have to understand the trait better to be sure. Just voicing an intuition..

Reply:
Well, `Author` is the hack.

But perhaps I've messed this up. Should the `identifier` be `did:rad:<root>` or something like that? Actually... I don't think `identifier` is even used anymore :thinking: I think I'll need to investigate that. I'm pretty sure `content_id` is the only thing needed.

Reply:
Ya, I was able to remove that :)

Reply:
Ah, great :+1: